### PR TITLE
Fix responsiveness visual issues - Closes #2963

### LINKED ISSUE #2963: Fix responsiveness visual issues
### Description
We have discovered several visual issues that occur on small screens. @ann-vashchenko has prepared design guides to resolve the said issues. This ticket aims to fix the responsiveness issues on small screens.

### Motivation
Improve accessibility on small screens.

### PR DESCRIPTION
**Pull request by isalga**
### What was the problem?
This PR resolves #2963

### How was it solved?

- Add `hideOnLargeViewPort` & `hideOnLargeViewPort` CSS mixins
- Changing flexbox grid viewport attributes
- Creating a custom doughnut chart tooltip to contain chart legends on medium size screens
- Trunkate Lisk address on medium viewport

### How was it tested?
Added test suite
Manual UI test on latest chrome browser

### CODE REVIEW

File: src/app/variables.css [hunk 1]
```diff
@@ -107,6 +107,8 @@ or "warn/action" ineastd of "red/green"
   --color-burnt-sienna: #ec6868;
   --color-ufo-green: #00a04b;
   --color-deep-green: #00d563;
+  --color-ufo-green: #2bd67b;
```
Comment (UsamaHameed) [inline src/app/variables.css:110]: Did you mean to name this something else? there is a `--color-ufo-green` already.

File: src/app/app.css [hunk 2]
```diff
@@ -156,6 +156,21 @@ html[data-useragent*='Linux'] {
...
+  }
+
+  html[data-useragent*='Windows'],
+  html[data-useragent*='Linux'] {
+    & :global(.expanded) {
+      & + .bodyWrapper {
+        padding-left: calc(var(--side-bar-shrunk-size));
+      }
+    }
+  }
```
Comment (UsamaHameed) [inline src/app/app.css:173]: Isn't the global.css file a better place for these styles?

File: src/components/shared/accountVisualWithAddress/accountVisualWithAddress.css [hunk 3]
```diff
@@ -1,3 +1,5 @@
+@import '../../../app/mixins.css';
```
Comment (UsamaHameed) [inline src/components/shared/accountVisualWithAddress/accountVisualWithAddress.css:1]: Please remove this if you don't plan to use any mixins here.

File: src/app/app.css (hunk 2 above)
Comment (isalga) [reply src/app/app.css:173]: I don't think so, because this is just affecting to the `bodyWrapper` at `app.js`

**Pull request closed without merging**